code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -26,6 +26,9 @@
 	if(mind)
 		miming = mind.miming
 
+	var/vox_chirp = list('sound/effects/vox/voxchirp1.ogg','sound/effects/vox/voxchirp2.ogg','sound/effects/vox/voxchirp3.ogg','sound/effects/vox/voxchirp4.ogg','sound/effects/vox/voxchirp5.ogg','sound/effects/vox/voxchirp6.ogg','sound/effects/vox/voxchirp7.ogg','sound/effects/vox/voxchirp8.ogg','sound/effects/vox/voxchirp9.ogg','sound/effects/vox/voxchirp10.ogg') //Credit to AlexTriceratops123  (https://www.youtube.com/watch?v=uA5yGyB_z5U) for the sound.
+	var/vox_hiss = list('sound/effects/vox/voxhiss1.ogg','sound/effects/vox/voxhiss2.ogg','sound/effects/vox/voxhiss3.ogg','sound/effects/vox/voxhiss4.ogg','sound/effects/vox/voxhiss5.ogg','sound/effects/vox/voxhiss6.ogg','sound/effects/vox/voxhiss7.ogg','sound/effects/vox/voxhiss8.ogg','sound/effects/vox/voxhiss9.ogg') //Credit to AlexTriceratops123  (https://www.youtube.com/watch?v=uA5yGyB_z5U) for the sound.
```

Comment:
Because these all share a similar name, you shouldn't hardcode them here. Instead do something like:
```dm
playsound(loc, 'sound/effects/vox/voxchirp[rand(1, 10)].ogg, ...)
```